projects/jsoncpp/json.proto

```diff
@@ -0,0 +1,81 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
this files probably needs another Google license header too -- please see the presubmit failure

Reply:
same for C++ files